src/cuminc.jl

```diff
@@ -0,0 +1,124 @@
+"""
+    fit(CumulativeIncidence, times, events, competing)
+
+Given a vector of times to events, a vector of indicators for the event of interest,
+and a vector of indicators for the competing event, compute the an estimate of the
+cumulative incidence of failure.
+
+The resulting `CumulativeIncidence` object has the following fields:
+
+* `times`: Distinct event times
+* `nevents`: Number of observed events of interest at each time
+* `ncensor`: Number of right censorings for the event of interest at each time
```

Comment:
Maybe call that `ncensored`? That's only two characters more.

Below, could also be `ncompeting`, though it's a bit longer so I'm not sure.